packages/backend-for-frontend/src/routers/notificationConfigRouter.ts

```diff
@@ -0,0 +1,37 @@
+import { ZodiosRouter } from "@zodios/express";
+import {
+  ZodiosContext,
+  ExpressContext,
+  zodiosValidationErrorToApiProblem,
+} from "pagopa-interop-commons";
+import { ZodiosEndpointDefinitions } from "@zodios/core";
+import { bffApi } from "pagopa-interop-api-clients";
+
+const notificationConfigRouter = (
+  ctx: ZodiosContext
+): ZodiosRouter<ZodiosEndpointDefinitions, ExpressContext> => {
+  const notificationConfigRouter = ctx.router(
+    bffApi.notificationConfigsApi.api,
+    {
+      validationErrorHandler: zodiosValidationErrorToApiProblem,
+    }
+  );
+
+  notificationConfigRouter
+    .get("/notification-configs/tenantNotificationConfigs", (_, res) =>
+      res.status(501).send()
+    )
+    .post("/notification-configs/tenantNotificationConfigs", (_, res) =>
+      res.status(501).send()
+    )
+    .get("/notification-configs/userNotificationConfigs", (_, res) =>
+      res.status(501).send()
+    )
+    .post("/notification-configs/userNotificationConfigs", (_, res) =>
+      res.status(501).send()
+    );
```

Comment:
Same here for `notificationConfigs` vs `notification-configs`. Here, it's weirder because we use two different standards in the same endpoints